app/models/katello/glue/candlepin/product.rb

```diff
@@ -7,12 +7,14 @@ def self.included(base)
       base.class_eval do
         lazy_accessor :productContent, :multiplier, :href, :attrs,
           :initializer => lambda { |_s| convert_from_cp_fields(Resources::Candlepin::Product.get(cp_id)[0]) }
-        # Entitlement Certificate for this product
-        lazy_accessor :certificate,
-          :initializer => lambda { |_s| Resources::Candlepin::Product.certificate(cp_id, self.organization.label) },
+        # Subscription for this product - used for certificate and key
+        lazy_accessor :subscription,
```

Comment:
Right, it is just `certificate` of the subscription, per https://github.com/Katello/katello/blob/master/app/lib/katello/resources/candlepin.rb#L653 (though it still contains lots of entitlement info). Renaming it to `product_certificate` since `product` variable is already used.